training/model_management/src/azureml/model/mgmt/downloader/download_utils.py

```diff
@@ -79,7 +79,9 @@ def _download_azure_artifacts(model_uri, model_dir):
         # https://github.com/Azure/azureml-assets/issues/283
         exit_code, stdout = run_command(download_cmd)
         if exit_code != 0:
-            raise Exception(f"Failed to download model files with URL: {model_uri}. Error => {stdout}")
+            raise Exception(
```

Comment:
This and the other changes in this file don't appear to exceed the 119 character limit. Why add line wrapping?

Reply:
reformatted with new length limit. Thanks Louie:)